# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/moz.build
# Commit: 88cd68e70502
# Full Hash: 88cd68e7050272bf76268c904feef7348d257b56
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1746104
# File Overlap Count: 1
# Description:
#   Bug 1746104 - part 6-2: Make constructors of `ContentCache::Selection` take `IMENotification::SelectionChangeData` or `WidgetQueryContentEvent` r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137428
# ==============================================================================

diff -r 2529c39ed6bf -r 88cd68e70502 widget/moz.build
--- a/widget/moz.build	Mon Feb 07 22:33:39 2022 +0000
+++ b/widget/moz.build	Mon Feb 07 22:33:40 2022 +0000
@@ -25,6 +25,9 @@
 with Files("*CompositorWidget*"):
     BUG_COMPONENT = ("Core", "Graphics")
 
+with Files("*ContentCache*"):
+    BUG_COMPONENT = ("Core", "DOM: UI Events & Focus Handling")
+
 with Files("*ContentData*"):
     BUG_COMPONENT = ("Core", "DOM: UI Events & Focus Handling")
 